lib/fastlane/actions/deploygate.rb

```diff
@@ -36,15 +36,15 @@ def self.run(params)
           Helper.log.info "DeployGate URL: #{Actions.lane_context[SharedValues::DEPLOYGATE_URL]}"
           Helper.log.info "Build successfully uploaded to DeployGate as revision \##{Actions.lane_context[SharedValues::DEPLOYGATE_REVISION]}!".green
         else
-          raise 'Error when trying to upload ipa to DeployGate'.red
+          fail 'Error when trying to upload ipa to DeployGate'.red
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.